cmd/promtool/main.go

```diff
@@ -59,6 +60,12 @@ func main() {
 		"The config files to check.",
 	).Required().ExistingFiles()
 
+	checkWebConfigCmd := checkCmd.Command("web-config", "Check if the web config files are valid or not.")
```

Comment:
Why web config is not part of Prometheus config? 🤔

Reply:
ok got my answer: https://github.com/prometheus/prometheus/pull/8316/files#diff-114e09a49c240c764f484c12b5f54019aafcfbe1a71f820d1c6b192ede1d5d18R13